src/Raven.Server/Web/System/ResourcesHandler.cs

```diff
@@ -54,7 +54,10 @@ public Task Resources()
 
                     writer.WriteStartArray();
                     var first = true;
-                    foreach (var dbDoc in ServerStore.StartingWith(context, "db/", GetStart(), GetPageSize()))
```

Comment:
GetPageSize() contains defaultValue, you should only pass int.Max there and it will work